static/build/.tmp_update/script/network/update_networking.sh

```diff
@@ -424,45 +424,64 @@ check_hotspotstate() {
 # This will work but it will not export the TZ var across all opens shells so you may find the hwclock (and clock app, retroarch time etc) are correct but terminal time is not.
 # It does set TZ on the tty that Main is running in so this is ok
 
-sync_time() {
-    if [ -f "$sysdir/config/.ntpState" ] && wifi_enabled; then
+check_ntpstate() {
+    if flag_enabled ntpState && wifi_enabled && [ ! -f "$sysdir/config/.hotspotState" ]; then
+        set_tzid
+        if [ -f /tmp/ntp_synced ] && [ ! "$is_booting" -eq 1 ]; then
```

Comment:
Why is the second condition necessary?

Reply:
Also, "NOT EQ 1" should just be "EQ 0"

Reply:
the check for is_booting allows the sync on boot

Reply:
`/tmp/ntp_synced` already doesn't exist on boot

Reply:
gotcha 👍